v19.2/string.md

```diff
@@ -6,22 +6,17 @@ toc: true
 
 The `STRING` [data type](data-types.html) stores a string of Unicode characters.
 
-## Aliases
+## Subtypes and aliases
 
-In CockroachDB, the following are aliases for `STRING`:
+<span class="version-tag">New in v19.2:</span> CockroachDB supports the following `STRING` subtypes for PostgreSQL compatibility:
 
-- `CHARACTER`
-- `CHAR`
-- `VARCHAR`
-- `TEXT`
+Subtype                                                               | Description                  
+----------------------------------------------------------------------|------------------------------
+`CHARACTER`, `CHARACTER(n)`, `CHAR`, `CHAR(n)`                        | Fixed-length                
+`CHARACTER VARYING`, `CHARACTER VARYING(n)`, `VARCHAR`, `VARCHAR(n)`  | Variable-length, with a limit  
+`TEXT`                                                                | Variable-length, with no limit
```

Comment:
i think the last two lines should look more like

```
`CHARACTER VARYING(n)`,  `VARCHAR(n)`                                 | Variable-length, with a limit  
`TEXT`, `VARCHAR`, `CHARACTER VARYING`, `STRING`                      | Variable-length, with no limit
```

(this looks correct on the docs for the other versions below)